LDM-294.tex

```diff
@@ -14,6 +14,7 @@
 \addtohist{3.4}{2017-12-21}{Org chart refresh - rules for review of epics \jira{DM-7501}}{W.~O'Mullane}
 \addtohist{3.5}{2018-06-18}{New DM product tree and revised org chart. Add deputy subsystem scientist. Approved in \jira{RFC-493}}{J.~Swinbank}
 \addtohist{3.6}{2019-01-29}{Review product tree. Add release management subsection. Update DMCCB and DMSE team. Approved in \jira{RFC-561}}{G.~Comoretto}
+\addtohist{3.7}{2019-02-06}{Add text for LSP and Middleware mangers. Time allocation policy for institutional scientists, add admin names to org chart. Approved in \jira{RFC-572}}{W.~O'Mullane,L.~Guy}
```

Comment:
Add space after comma between names.

Reply:
ok .. but you want anew pdf in docushare then ?